packages/main/src/FileUploader.ts

```diff
@@ -303,7 +336,17 @@ class FileUploader extends UI5Element implements IFormInputElement {
 	}
 
 	_onChange(e: Event) {
-		const changedFiles = (e.target as HTMLInputElement).files;
+		let changedFiles = (e.target as HTMLInputElement).files;
+
+		const validFileSizes = this._validateFileSize(changedFiles);
```

Comment:
Boolean var names should have the adjective/participle at the end, f.e. `taskDone`, `nameValid`, etc. therefore this should be `fileSizesValid`, meaning "file sizes are valid". When it ends with a noun (`validFileSizes`) this means it's an array of valid file sizes.

Reply:
Function no longer returns a boolean